examples/existing_vpc/variables.tf

```diff
@@ -9,10 +9,17 @@ variable "region" {
   type        = string
   default     = "us-south"
 }
+
 variable "vpc_id" {
   description = "The ID of the VPC where the VSI will be created."
   type        = string
 }
+
+variable "public_gateway_name" {
+  description = "The name of the public gateway"
+  type        = string
+}
```

Comment:
Instead of adding a new variable, can you just hard code the variable values in the main.tf of the existing_vpc example main.tf ?